chi_edge/cli.py

```diff
@@ -11,13 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import os
 
+import chi
 import click
+import json
+import os
+import re
 import yaml
 
 from chi_edge import SUPPORTED_DEVICE_TYPES, VIRTUAL_SITE_INTERNAL_ADDRESS, ansible
 
+from chi_edge.vendor.FATtools import cp
+
+
+from keystoneauth1 import adapter, session
```

Comment:
These are not used as far as I can tell

Reply:
The `re` and `keystoneauth1.session` modules are also not used